pepper-apis/src/test/java/org/broadinstitute/ddp/service/PdfGenerationServiceTest.java

```diff
@@ -66,16 +63,17 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(PdfGenerationServiceTest.class);
 
-    private PdfTestingUtil.PdfInfo pdfInfo;
+    private static PdfTestingUtil.PdfInfo pdfInfo;
+
     private PdfGenerationService pdfGenerationService = new PdfGenerationService();
 
-    @Before
-    public void setup() throws Exception {
+    @BeforeClass
+    public static void setup() throws Exception {
         pdfInfo = PdfTestingUtil.makePhysicianInstitutionPdf();
     }
```

Comment:
We can reuse `pdfInfo` so let's make it static instead of instantiating it for every test.